src/components/ToolBar/ToolBar.svelte

```diff
@@ -12,9 +14,28 @@
 
   export let version;
   export let id = Maybe.None();
+  export let inputLanguage = 'fi';
+  export let energiatodistusKieli = Maybe.None();
+
+  let bilingual = true;
+  let selectedLanguage = 'fi';
+
+  $: bilingual = R.compose(
+    Maybe.orSome(true),
+    R.map(R.equals(et.kielisyys.bilingual))
+  )(energiatodistusKieli);
+
+  $: inputLanguage = bilingual ?
+      selectedLanguage : energiatodistusKieli.map(et.kielisyysKey).some();
```

Comment:
eikö tässä olisi turvallisempaa tehdä näin
`Maybe.orSome(selectedLanguage, R.map(et.kielisyysKey, energiatodistusKieli))`

Reply:
joo - muutan noin